packages/shared-components/src/SelectionQuestion.tsx

```
@@ -0,0 +1,78 @@
+      <div className="flex-1">
+        {content !== '<br>' && (
+          <div
+            className={twMerge(
+              'bg-primary-10 prose prose-p:!m-0 prose-img:!m-0 mb-4 min-h-[6rem] max-w-none flex-initial rounded border border-slate-300 p-4 leading-6'
+            )}
+          >
+            <Markdown content={content} />
+          </div>
+        )}
+
+        {evaluation && evaluation.explanation && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)